CONTRIBUTING.md

```diff
@@ -1,27 +1,35 @@
-# Contributing a Devfile Stack to this registry
+# Contributing to this registry
 
-This document outlines the requirements for contributing a devfile stack to this repository
+This document outlines the requirements for contributing a devfile stack or sample to this repository
 
 The [devfile registry structure](https://github.com/devfile/api/blob/main/docs/proposals/registry/registry-structure.md#repository-structure) design doc provides some useful background information on the structure of resources in a devfile registry (and its Git repository).
 
 ## Prerequisites
 
-The following are required to build the devfile index container image containing your stack:
+The following are required to build the devfile index container image containing your stack or sample:
```

Comment:
I think passive voice here reads fine. If we wanted to be extra user focused, we could make it active voice:

`To build the devfile index container image containing your stack or sample, you must complete the following:`

Reply:
You must `install the following` rather than `complete the following`? Since it's highlighting the tools we need to install?

Reply:
Sure, `must install the following` sounds great!